src/notices/faq/components/FAQContent.tsx

```diff
@@ -0,0 +1,126 @@
+import Image from "next/image";
+import React, { useMemo, useState } from "react";
+import { FAQ_CONTENTS } from "../constants";
+
+const FAQContent = () => {
+  const CONTENT_COUNT = 5;
+  const [currentPage, setCurrentPage] = useState(1);
+  const [isOpen, setIsOpen] = useState(["q-1"]);
```

Comment:
[Lv2] state 내용물이 배열 형태라면 변수명이 is~ 로 시작하는건 헷갈리기 쉬울것같습니다 (보통 is~ 로 시작하는건 boolean 타입이기 때문)

openState 같은 형식으로 바꾸는건 어떨까요?

Reply:
[Lv2] 배열 내부 데이터가 q-1 같은 string 형식인 이유가 있을까요? 받은 데이터의 개수가 명확하기 때문에 array index로 구분하여 boolean으로 관리해도 좋을것같습니다

```suggestion
  const [openState, setOpenState] = useState(Array(CONTENT_COUNT).fill(false));
```
이렇게 하면 open 여부를 string으로 비교하는게 아닌 true/false로 바로 사용할수있기 때문에 코드 가독성 면에서도 유리합니다

Reply:
아 인덱스로 바로 접근이 가능하겠군요!! 감사합니다😀